### PR TITLE
add note about direnv allow

### PR DESCRIPTION
**Pull request by arthurwolf**
## Checklist

 - [ ] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [ ] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -69,6 +69,14 @@ and Direnv installed.
 This folder contains another `.envrc` file that adds all the binaries needed to
 build the docs to `$PATH`.
 
+In short, when you `cd` into this folder, you should see this message:
+
+```sh
+direnv: error wire-server/docs/.envrc is blocked. Run `direnv allow` to approve its content
+```
+
+Run `direnv allow` to allow the `.envrc` file to modify your environment, and you should then have all the binaries/correct environment needed to build the docs.
```
Comment (supersven) [inline docs/README.md:78]:
```suggestion
Run `direnv allow` to allow the `.envrc` file to modify your environment. Then, you should have everything (binaries, environment variables) needed to build the docs.
```

**Review by supersven — APPROVED** [review]
Additionally, I think it would be good to add links to the `direnv` and `nix` websites (https://github.com/wireapp/wire-server/pull/2845/files#diff-0b5ca119d2be595aa307d34512d9679e49186307ef94201e4b3dfa079aa89938R66).

File: docs/README.md (hunk 1 above)
Comment (elland) [reply docs/README.md:78]: Should we mention cachix here as well? Without it set up, `direnv allow` will take hours to complete.

Comment (arthurwolf) [reply docs/README.md:78]: @elland do you have a link/recommendation of what to say about cachix?

Comment (elland) [reply docs/README.md:78]: It's mentioned here https://github.com/wireapp/wire-server/blob/develop/docs/src/how-to/install/dependencies.rst :)

**Pull request merged**